src/frequenz/sdk/microgrid/_power_distributing/_component_status/_ev_charger_status_tracker.py

```diff
@@ -115,7 +115,7 @@ def _handle_ev_data(self, ev_data: EVChargerData) -> ComponentStatusEnum:
 
         if self._is_working(ev_data):
             if self._last_status == ComponentStatusEnum.NOT_WORKING:
-                _logger.warning(
+                _logger.info(
                     "EV charger %s is in WORKING state.",
```

Comment:
Wouldn't it make sense to also say it's becoming functional after being not working? Or at least say "changed state to WORKING` or something like that, saying more explicit it is a state change?